packages/@lwc/engine-core/src/framework/html-element.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+// This is a temporary workaround to get the @lwc/engine-server to evaluate in node without having
+// to inject at runtime.
```

Comment:
This comment is not accurate any more as we are now injecting the APIs. The `HTMLElement` is one of the injected APIs. It's something we can probably refactor in a follow-up PR.